util/calivppctl.go

```diff
@@ -0,0 +1,1089 @@
+package main
```

Comment:
This probably needs a `gofmt -w .`
Also, can you prepend it with a license header ?
Last can we rename this to `cmd/calicovppctl/main.go` ?